doc/releases/release-notes-4.0.rst

```diff
@@ -253,6 +307,21 @@ Boards & SoC Support
 
   * Added Thingy53 forwarding of network core pins to network core for SPI peripheral (disabled
     by default) including pin mappings.
+  * linkserver: Added .elf support to the linkserver flash command
+  * pyocd: Add support for .elf files to the pyocd flash command
+  * jlink: Added support for .elf files to the jlink flash command
```

Comment:
As already mentioned, this needs to be removed.
https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/releases/release-notes-4.0.rst?plain=1#L272

Please try to re-focus your PR on changes to NXP platforms, not just any commit from NXP as this unmanageable otherwise